realm/public.gno

```diff
@@ -166,27 +166,21 @@ func GetThreadPosts(userPostsAddr std.Address, threadID int, replyID int, startI
 
 // Get home posts for a user, which are the user's top-level posts plus all posts of all
 // users being followed.
-// The response is a map of postTime -> []*Post where the post time as a string like
-// "2024-02-26 13:02:46 +0000 UTC m=+1708952566.000000001". (The avl.Tree sorts by the post time.)
-// The value is an array of *Post because there can be multiple posts with the same time.
+// The response is a map of postTime/postID -> *Post where the post time as a string like
+// "2024-02-26 13:02:46 +0000 UTC m=+1708952566.000000001". The key includes the postID because
```

Comment:
Can we just use the ID for the key, the date is useless? The date is in the Post struct I think.

Reply:
The ID already sorts the messages by order.

Reply:
Good point. I was thinking that, in the future, we may want to limit the results by date. But you're right, we can look at the date in the Post struct. I'll push a commit to sort only by ID.